publish_dnn_model_services_test.go

```diff
@@ -179,191 +104,36 @@ func dnnmodelServiceIsRunning() (err error) {
-			"--transact",
-			"-y",
-		},
-	}
-
-	err = runCommand(command)
-
-	if err != nil {
-		return
-	}
-
-	command = ExecCommand{
-		Command:   "snet",
-		Directory: dnnModelServicesDir,
-		Args: []string{
-			"contract",
-			"MultiPartyEscrow", "--at", multiPartyEscrow,
-			"deposit", "1000000",
-			"--transact",
-			"-y",
-		},
-	}
-
-	err = runCommand(command)
-
-	if err != nil {
-		return
-	}
+func dnnmodelMakeACallUsingPaymentChannel(table *gherkin.DataTable) (err error) {
 
-	output := dnnModelServicesDir + "/expiration.txt"
-
-	command = ExecCommand{
-		Command:    "snet",
-		Directory:  dnnModelServicesDir,
-		Args:       []string{"mpe-client", "block_number"},
-		OutputFile: output,
-	}
-
-	err = runCommand(command)
-
-	if err != nil {
-		return
-	}
-
-	expirationText, err := readFile(output)
-	if err != nil {
-		return
-	}
-
-	expiration, err := strconv.Atoi(strings.TrimSpace(expirationText))
-
-	if err != nil {
-		return
-	}
-
-	expiration += 12000
-
-	command = ExecCommand{
-		Command:   "snet",
-		Directory: dnnModelServicesDir,
-		Args: []string{
-			"contract",
-			"MultiPartyEscrow", "--at", multiPartyEscrow,
-			"openChannel", organizationAddress,
-			"420000", strconv.Itoa(expiration), "0",
-			"--transact",
-			"-y",
-		},
-	}
-
-	err = runCommand(command)
-
-	return
-}
-
-func dnnmodelCompileProtobuf() (err error) {
-
-	command := ExecCommand{
-		Command:   "snet",
-		Directory: dnnModelServicesDir,
-		Args: []string{
-			"mpe-client",
-			"compile_from_dir",
-			envSingnetRepos + "/dnn-model-services/Services/gRPC/Basic_Template/service/service_spec",
-			"0",
-		},
-	}
-
-	err = runCommand(command)
-
-	return
-}
-
-func dnnmodelMakeACallUsingPaymentChannel() (err error) {
-
-	outputFile := dnnModelServicesDir + "/output.txt"
-
-	fileContains := checkFileContains{
-		output:     outputFile,
-		strings:    []string{organizationAddress, "420000"},
-		ignoreCase: true,
-	}
-
-	command := ExecCommand{
-		Command:   "snet",
-		Directory: dnnModelServicesDir,
-		Args: []string{
-			"mpe-client",
-			"print_my_channels", multiPartyEscrow,
-		},
-		OutputFile: outputFile,
-	}
-
-	err = runCommand(command)
-
-	ok, err := checkFileContainsStrings(fileContains)
-	err = fileContainsError(fileContains, ok, err)
-
-	if err != nil {
-		return
-	}
-
-	for i := 0; i < 3; i++ {
-		command = ExecCommand{
-			Command:   "snet",
-			Directory: dnnModelServicesDir,
-			Args: []string{
-				"mpe-client",
-				"call_server", multiPartyEscrow,
-				"0", "10", "localhost:8090",
-				"--service", "Addition", "add", `{"a":10,"b":32}`,
-			},
-		}
-
-		err = runCommand(command)
-		if err != nil {
-			return
-		}
-
-	}
+	name := getTableValue(table, "name")
+	organization := getTableValue(table, "organization name")
+	daemonPort := getTableValue(table, "daemon port")
 
-	return
+	cmd := NewCommand().Dir(dnnModelServicesDir)
+	cmd.
+		Run("snet client balance --snt %s --multipartyescrow %s", singnetTokenAddress, multiPartyEscrow).
```

Comment:
I believe @astroseger mentioned that there is a way to not pass addresses of contracts manually each time.

Reply:
It requires the following steps:
- migrate contracts
- get contract addresses and put the in networks folder in snet-cli
- rebuilding snet-cli
It seems that in this case running test and building process should be mixed.
I prefer to postpone this investigation to a separate pull request.